flink-scala/src/main/scala/org/apache/flink/api/scala/GroupedDataSet.scala

```diff
@@ -356,6 +357,40 @@ class GroupedDataSet[T: ClassTag](
   }
 
   /**
+    * Applies a special case of a reduce transformation `maxBy` on a grouped [[DataSet]]
+    * The transformation consecutively calls a [[ReduceFunction]]
+    * until only a single element remains which is the result of the transformation.
+    * A ReduceFunction combines two elements into one new element of the same type.
+    *
+    * @param fields Keys taken into account for finding the minimum.
+    * @return A [[ReduceOperator]] representing the minimum.
```

Comment:
The return type is not correct. You can remove `@param` and `@return` as these are not used for the other methods in this class either.